src/patternfly/components/Nav/nav.scss

```diff
@@ -572,36 +588,63 @@
   }
 }
 
-// Scroll button
+.pf-c-nav__list,
+.pf-c-nav__simple-list,
+.pf-c-nav__section {
+  flex: 1 0 100%;
+}
+
+// Scroll buttons
 .pf-c-nav__scroll-button {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  z-index: var(--pf-c-nav__scroll-button--ZIndex);
-  display: var(--pf-c-nav__scroll-button--Display);
-  align-items: center;
-  justify-content: center;
+  flex: none;
   width: var(--pf-c-nav__scroll-button--Width);
-  height: 100%;
-  padding-right: var(--pf-c-nav__scroll-button--PaddingRight);
-  padding-left: var(--pf-c-nav__scroll-button--PaddingLeft);
-  visibility: var(--pf-c-nav__scroll-button--Visibility);
+  color: var(--pf-c-nav__scroll-button--Color);
   background-color: var(--pf-c-nav__scroll-button--BackgroundColor);
   border: 0;
+  outline-offset: var(--pf-c-nav__scroll-button--OutlineOffset);
+  opacity: 0;
```

Comment:
why do you need the opacity here? could you not just display none on the icon arrow

Reply:
That kills any animation. By allowing react to set `disabled` on the scroll buttons when not scrollable, they become untabbable/unfocusable.